pkg/restore/restore_test.go

```diff
@@ -2340,21 +2339,15 @@ func assertRestoredItems(t *testing.T, h *harness, want []*test.APIResource) {
 			}
 
 			res, err := client.Get(context.TODO(), item.GetName(), metav1.GetOptions{})
-			if !assert.NoError(t, err) {
-				continue
-			}
+			require.NoError(t, err)
```

Comment:
I'm a little confused here.
Does the original code want the testing flow going on even meeting an error?
If so, it seems the change here seems not right.